modules/channel-web/src/views/lite/components/Composer.tsx

```diff
@@ -162,6 +162,7 @@ export default inject(({ store }: { store: RootStore }) => ({
   composerLocked: store.composer.locked,
   composerHidden: store.composer.hidden,
   composerPlaceholder: store.composer.composerPlaceholder,
+  composerMaxTextLength: store.composer.composerMaxTextLength,
   updateMessage: store.composer.updateMessage,
```

Comment:
Do we already have backend validation for text length? Ideally we would have validation in the backend as well otherwise it's kind of incomplete.